src/Native/Windows/clrcompression/CMakeLists.txt

```diff
@@ -74,3 +74,7 @@ GENERATE_EXPORT_HEADER( clrcompression
 )
 
 install (TARGETS clrcompression DESTINATION .)
+
+install (FILES ${CMAKE_CURRENT_BINARY_DIR}/Debug/clrcompression.pdb DESTINATION .
+    CONFIGURATIONS Debug
```

Comment:
> Do you think that we should have a pdb for release builds as well? I'm not familiar enough with native builds to know when we want one and when we can get one.

We should always be producing a `.pdb` for any build we do (either Debug or Release).  If we aren't for release, then we need to get our CMake rules for this thing updated.  @ianhays Did we go out of our way to not produce a PDB for System.IO.Compression when building Release?

Not having a PDB will be a ship blocker (today my understanding is the version of this library that we produce is still built out of TFS and doesn't use CMake).